src/Features/Core/Portable/Diagnostics/Analyzers/NamingStyles/NamingStylePreferencesInfo.cs

```diff
@@ -33,5 +40,16 @@ internal bool TryGetApplicableRule(ISymbol symbol, ISymbolCategorizationService
             applicableRule = matchingRule.GetBestMatchingRule(symbol, categorizationService);
             return true;
         }
+
+        private bool IsSymbolNameAnalyzable(ISymbol symbol)
+        {
+            var methodSymbol = symbol as IMethodSymbol;
+            if (methodSymbol != null && methodSymbol.MethodKind != MethodKind.Ordinary)
```

Comment:
Are there things that should not be analyzable for things that are not methods?  Explicitly implemented properties are the only ones that come to mind quickly.